### PR TITLE
Fix link vonoroi

### PR DESCRIPTION
**Pull request by paulitapb**
I fixed the link in the reference section. The link was right but was missing some parts so you couldn't click on it. I also added some latex formatting because the previous notation, which was italics,  just looked like plain text.

### CODE REVIEW

**Review by dschult — APPROVED** [review]
Also looks great!  Thanks!

File: networkx/algorithms/voronoi.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def voronoi_cells(G, center_nodes, weight="weight"):
     G : NetworkX graph
 
     center_nodes : set
-        A nonempty set of nodes in the graph `G` that represent the
+        A nonempty set of nodes in the graph $G$ that represent the
```
Comment (rossbar) [inline networkx/algorithms/voronoi.py:28]: Technically `` `G` `` is correct according to the [numpydoc docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#parameters): function parameters are supposed to be enclosed in single backticks when referenced elsewhere in the docstring.

**Review by rossbar — COMMENTED** [review]
Generally LGTM, just one comment about reverting the changes to `G`

File: networkx/algorithms/voronoi.py (hunk 1 above)
Comment (paulitapb) [reply networkx/algorithms/voronoi.py:28]: Good to know! I will revert it

Comment (paulitapb) [reply networkx/algorithms/voronoi.py]: It's done

**Review by rossbar — APPROVED** [review]
Thanks @paulitapb !

**Pull request merged**